logstash-core/spec/logstash/util/wrapped_synchronous_queue_spec.rb

```diff
@@ -95,25 +91,32 @@ def poll(*) shift(); end
 
         it "appends batches to the queue" do
           batch = write_client.get_new_batch
-          5.times {|i| batch.push(LogStash::Event.new({"message" => "value-#{i}"}))}
+          messages = []
+          8.times do |i|
```

Comment:
@jordansissel ok this is a really stupid thing now (it was a dumb change, to begin with :P). There were tests above this one for `take` and `offer(timeout)` that are now gone but for which the `8` worked as a size without more code changes.
Will revert :)